model-api/src/commonMain/kotlin/org/modelix/model/api/IEnum.kt

```diff
@@ -0,0 +1,18 @@
+package org.modelix.model.api
+
+/**
+ * Superinterface for enums generated by model-api-gen
+ */
+interface IEnum {
```

Comment:
The name `IEnum` is a bit too generic. Maybe `IPropertyValueEnum`?

Reply:
Sure, I also moved the interface from model-api to model-api-gen-runtime, because i think it makes more sense there.